.github/workflows/compile.yaml

```diff
@@ -20,11 +20,19 @@ jobs:
           java-version: 11
       - name: Setup Gradle
         uses: gradle/gradle-build-action@v2
-
       - name: Execute Gradle build
         run:  |
               if [ "$RUNNER_OS" == "Windows" ]; then
                 export HADOOP_HOME="$(pwd)/.github/workflows/hadoop3-win-binaries"
               fi
               ./gradlew build testNative --no-daemon
-        shell: bash
\ No newline at end of file
+        shell: bash
+      - name: Set up QEMU
+        if: runner.os == 'Linux'
+        uses: docker/setup-qemu-action@v2
+      - name: Build and push docker image
+        if: runner.os == 'Linux'
+        run: |
+            docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think you should run also 
```
./gradlew build -Dquarkus.container-image.build=true
```

Reply:
it should build the image anyway because of the properties of build task in the gradle settings